catalogs/catalog-hadoop/src/main/java/org/apache/gravitino/catalog/hadoop/authentication/kerberos/KerberosClient.java

```diff
@@ -67,7 +67,8 @@ public String login(String keytabFilePath) throws IOException {
     UserGroupInformation.setConfiguration(hadoopConf);
     UserGroupInformation kerberosLoginUgi =
         UserGroupInformation.loginUserFromKeytabAndReturnUGI(catalogPrincipal, keytabFilePath);
-    UserGroupInformation.setLoginUser(kerberosLoginUgi);
+    // Replace setLoginUser with an alternative approach
```

Comment:
There no real need for this comment in teh code, this wouddl be better placed as a version control comment when making the change.

Reply:
Should I remove the comment and update the PR?
@justinmclean

Reply:
If you can please that would help

Reply:
Made the changes accordingly please review it!!